python/ccxt/async_support/southxchange.py

```diff
@@ -330,3 +345,26 @@ def sign(self, path, api='public', method='GET', params={}, headers=None, body=N
                 'Hash': self.hmac(self.encode(body), self.encode(self.secret), hashlib.sha512),
             }
         return {'url': url, 'method': method, 'body': body, 'headers': headers}
+
+    async def fetch_ohlcvs(self, symbol, timeframe='1m', since=None, limit=500, params={}):
```

Comment:
it better to leave default limit=None and add in function
if limit is None:
  limit = 500
To avoid passing None from above